biggraphite/cli/command_list.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# Copyright 2016 Criteo
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""List Command."""
+
+from __future__ import print_function
+
+from biggraphite.cli import command
+
+
+def list_metrics(accessor, pattern):
+    """Return the list of metrics corresponding to pattern. Exit with error message if None.
+
+    Args:
+        - pattern, string, e.g. my.metric.a or my.metric.**.a
+    """
+    accessor.connect()
```

Comment:
Yep, each command should start by doing a connect() if they are going to use the accessor. Some commands don't need to call connect() because they don't use it